dev/io.openliberty.microprofile.reactive.messaging.internal/resources/io/openliberty/microprofile/reactive/messaging/internal/ReactiveMessagingProvider.nlsprops

```diff
@@ -0,0 +1,32 @@
+#CMVCPATHNAME N/A
+#COMPONENTPREFIX CWMRX
+#COMPONENTNAMEFOR CWMRX MicroProfile Reactive
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#ISMESSAGEFILE true
+# #########################################################################
+###############################################################################
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# This file follows the WebSphere Message Guidelines.
+# For more information, visit:
+# http://washome.austin.ibm.com/xwiki/bin/view/MessagesTeam/
+#
+
+#-----------------------------------------------------------------------------------------------------------------------------
+# CWMRX   0000-0999       MicroProfile Reactive Streams
+# CWMRX   1000-1999       MicroProfile Reactive Messaging
+#-----------------------------------------------------------------------------------------------------------------------------
+
+missing.context.service.CWMRX1200E=CWMRX1200E: The {0} channel is configured to use the {1} context service, but no such context service was found. Ensure that the context service is defined in the server.xml and that the concurrent feature is enabled.
```

Comment:
```suggestion
missing.context.service.CWMRX1200E=CWMRX1200E: The {0} channel is configured to use the {1} context service, but no such context service was found. Ensure that the context service is defined in the server.xml file and that the concurrent feature is enabled.
```

Reply:
Is it clear to the developer what "the concurrent feature" refers to? I'm assuming its some `mpReactiveMessaging-*.*` but not sure what "concurrent" means in this context

Reply:
or does it mean one of the  `concurrent-*.*` features? If so, should use "Jakarta Concurrency feature"

Reply:
It's the latter, I'll update it - though it could also be the Java EE concurrency feature, depending on which version of mpReactiveMessaging the user is running.